src/Guesser/TypeGuesser.php

```diff
@@ -61,8 +59,8 @@ public function guessType($class, $property, ModelManagerInterface $modelManager
             case 'float':
                 return new TypeGuess('number', [], Guess::MEDIUM_CONFIDENCE);
             case 'integer':
-            case 'bigint':
```

Comment:
Can we remove this? IMHO we should keep support for this and throw a deprecation warning instead

Reply:
Ok, I'll add them just in case, but apparently [those type don't exist](https://github.com/doctrine/mongodb-odm/blob/master/lib/Doctrine/ODM/MongoDB/Types/Type.php#L26).

Reply:
Thanks, I agree with @core23 here, as this could lead to a weird behavior + it’s an easy way to deprecate the options 👍🏻